.env.example

```diff
@@ -18,3 +18,9 @@ ROUTE53_ZONEID=XXXXXXXXX
 # The name of the S3 bucket where the container images that gitpod creates are stored
 # If there is no value we create a new bucket with the name "container-registry-<cluster name>-<account ID>"
 CONTAINER_REGISTRY_BUCKET=
+
+
+# Apply the given policy as Permissions Boundary to all Roles and Users in the scope
+# https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html
+# Default: no boundaries
+PERMISSIONS_BOUNDARY_ARN=
```

Comment:
Do we need to document why / how someone would use this in the README? If so lets just file an issue to track it. I don't think it needs to block.